OSDOCS-16941: Fix DITA CalloutList in OLM modules (batch 1) (#115365)
openshift/openshift-docs: modules/olm-software-catalog-architecture.adoc

metadata:
  name: cluster
spec:
  disableAllDefaultSources: true
  sources: [
    {
      name: "community-operators",
      disabled: false
    }
  ]
----

> +
> where:
> 
> `spec.disableAllDefaultSources`:: Specifies an override that controls availability of all default catalogs that are configured by default during an {product-title} installation.
> `spec.sources`:: Specifies disabling default catalogs individually by changing the `disabled` parameter value per source.

Reviewer: ```suggestion

** The `disableAllDefaultSources` field is an override that controls availability of all default catalogs that are configured by default during an {product-title} installation.
** Disable default catalogs individually by changing the `disabled` parameter value per source.
```
